tests/test_models.py

```diff
@@ -312,6 +312,26 @@ def test_opening_gain_ref(tmp_path):
     assert isinstance(gain, datamodels.GainRefModel)
 
 
+# Gain tests
+def test_make_ipc():
+    ipc = utils.mk_ipc(shape=(20, 20))
```

Comment:
Let's do (3, 3).

Reply:
This should remain a shape different from default to better test the code. The default shape is tested in test_opening_ipc_ref

Reply:
Can we at least make it odd?  With an even convolution kernel shape I feel like we also need to define how it's centered.